src/components/ContactComponent/ContactComponent.tsx

```diff
@@ -1,23 +1,47 @@
 import React from 'react'
 import './ContactComponent.scss'
 import { Helmet } from 'react-helmet'
+import ContactForm from "../ContactFormComponent/ContactFormComponent";
+import { FacebookOutlined, GithubOutlined, GlobalOutlined } from "@ant-design/icons/lib";
+import { codrillWebsite, facebookProfile, githubProfile } from "../../constants/urls";
 
 const Contact = () => {
-  return (
-    <div className="contact">
-  
-      <Helmet>
-        <title>CConverter - Kontakt</title>
-      </Helmet>
-      
-      <div className="contact__form">
-        Formularz kontaktowy
-      </div>
-      <div className="contact__details">
-        Dane kontaktowe
-      </div>
-    </div>
-  )
+    return (
+        <div className="contact">
+
+            <Helmet>
+                <title>CConverter - Kontakt</title>
+            </Helmet>
+
+            <div className="contact__details">
+                <p>Znajdziesz nas na:</p>
+
+                <div className="wrapper">
```

Comment:
is this class predefined in ant design somehow? if no, then please consider using something less general. 
Class names like this are good solution, when they are defined globally, have some generic styles, and are used across whole application. Otherwise it can be a bit confusing.

Reply:
Good point. Changed